src/components/Shared/PerfCompareHeader.tsx

```diff
@@ -66,23 +39,23 @@ function PerfCompareHeader(props: PerfCompareHeaderProps) {
         >
           {strings.title}
         </Typography>
-        <Typography
-          component='div'
-          align='center'
-          gutterBottom
-          className='perfcompare-tagline'
-        >
-          {strings.tagline}
-        </Typography>
-        <Button className='learn-more-btn'>{strings.learnMore}</Button>
+        {view === 'search' && (
+          <Typography
+            component='div'
+            align='center'
+            gutterBottom
+            className='perfcompare-tagline'
+          >
+            {strings.tagline}
+          </Typography>
+        )}
+        {view == 'search' && (
+          <Button className='learn-more-btn'>{strings.learnMore}</Button>
+        )}
```

Comment:
What do you think of gathering the 2 items in the same block?
```suggestion
        {view === 'search' && (
          <>
            <Typography
              component='div'
              align='center'
              gutterBottom
              className='perfcompare-tagline'
            >
              {strings.tagline}
            </Typography>
            <Button className='learn-more-btn'>{strings.learnMore}</Button>
          </>
        )}
```
I think this makes the flow easier to read.

Reply:
Agreed. I rushed through this one so thanks for catching it.